### ISSUE TITLE
collectFormValues fails in IE9

### ISSUE THREAD

**Issue by kembuco** [ansman/validate.js#39]
The collectFormValues function fails in IE9 as the dataset attribute is not supported.  The code fails with the following error:

```
SCRIPT5007: Unable to get value of the property 'ignored': object is null or undefined
```

The offending code:
```
if (input.dataset.ignored !== undefined) {
    continue;
}
```

**Comment by ansman** [comment]
Released as `0.6.1`

**Comment by ansman** [comment]
Released as `0.6.1`
